src/shared/components/case-editor/case-edit-confirm/case-edit-confirm.component.spec.ts

```diff
@@ -82,6 +97,16 @@ describe('CaseEditConfirmComponent', () => {
     expect(de.nativeElement.textContent).toBeDefined();
     expect(de.nativeElement.textContent.trim()).toEqual('A body with mark down');
   });
+
+  it('should show valid title on the page', () => {
+    const title = component.getCaseTitle();
+    expect(title).toEqual('# 1234567812345678: test');
+  });
+
+  it('should show valid title on the page', () => {
```

Comment:
This test does not look like its testing for a title

Reply:
changed the test description